<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wally::qtfront::MainWindow</class>
 <widget class="QMainWindow" name="wally::qtfront::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="TreeItemView" name="treeItemView" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="EditorView" name="tabView" native="true"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="AppCmdView" name="cmdView" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TreeItemView</class>
   <extends>QWidget</extends>
   <header>qtfront/tree_item_view.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AppCmdView</class>
   <extends>QWidget</extends>
   <header>qtfront/app_cmd_view.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EditorView</class>
   <extends>QWidget</extends>
   <header>qtfront/editor_view.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
